__tests__/components/BB_test.js

```diff
@@ -95,7 +95,7 @@ describe("BB", () => {
   });
 
   it("has a sortBy selector", () => {
-    expect(shallow_BB().find("#sortBySelector").length).toEqual(1);
+    expect(mounted_BB().find("#sortBySelector").length).toEqual(2);
```

Comment:
why did this change to 2? mount vs shallow?

Reply:
not sure - I chalked it up to the issue where enzyme sometimes returns multiple copies of the same thing.